Velocikey Configuration Pt. 1 (#7148)
qmk/qmk_firmware: docs/feature_velocikey.md

 As long as Velocikey is enabled, it will control the speed regardless of any other speed setting that your RGB lights are currently on.

 ## Configuration

>  You are able to configure the top typing speed that Velocikey considers the

Reviewer: Some wordsmithing here; I think this reads better.

```suggestion
 You are able to configure the typing speed that Velocikey considers to be the maximum with the define below. However, the decay rate is currently not configurable.
```

>  fastest with the define below. However, the decay rate is currently not

Reviewer: ```suggestion
```

>  configurable.

Reviewer: ```suggestion

```
